.rosinstall

```diff
@@ -3,3 +3,11 @@
 # IF YOU WANT TO CHANGE THE ROS ENVIRONMENT VARIABLES
 # USE THE rosinstall TOOL INSTEAD.
 # IF YOU CHANGE IT, USE rosinstall FOR THE CHANGES TO TAKE EFFECT
+- setup-file: {local-name: /opt/ros/kinetic/setup.sh}
```

Comment:
Could we add a few comments here explaining what `setup-file` and `git` entries are doing? (Alternatively, we could add a link to the documentation)

Reply:
Done